### PR TITLE
Allowing extraEnv to be added to agentservices and agent deployments.

### PR DESCRIPTION
**Pull request by lucacerone**
Simple strategy to inject environment variables to the "agentservices", and "agent" deployments
as discussed in issue: https://github.com/allegroai/clearml-helm-charts/issues/75

I am not sure whether I should have bumped the chart version, or whether that would be automatic.
Also, I am not sure whether the K8s Glue Agent `k8sagent` might also need extra environment variables
and it would be worth to add an `extraEnvs` to the values and to the deployment template.

### CODE REVIEW

**Comment by valeriano-manassero** [comment]
Hi @lucacerone and ty for the PR. Pls bump up chart version and regenerate README.md with helm-docs. After that we can start CI tests!

**Comment by lucacerone** [comment]
@valeriano-manassero thanks a lot! I bumped the patch and regenerated the README.md

File: charts/clearml/values.yaml [hunk 1]
```diff
@@ -281,6 +281,8 @@ agentGroups:
       repository: "ubuntu"
       pullPolicy: IfNotPresent
       tag: "18.04"
+
```
Comment (valeriano-manassero) [inline charts/clearml/values.yaml:284]: Lint is not passing due trailing spaces, can you fix it pls?

**Review by valeriano-manassero — CHANGES_REQUESTED** [review]

File: charts/clearml/values.yaml (hunk 1 above)
Comment (lucacerone) [reply charts/clearml/values.yaml]: Fixed it. Sorry I run `helm lint chars/clearml` but it didn't report any error.

**Review by valeriano-manassero — APPROVED** [review]

**Comment by valeriano-manassero** [comment]
PR lgtm but, for some reason, Chart installation is failing; I'm not sure it's related PR itself but I should check in deep with a local install.
Sorry but I need to ask you to wait some day so I can test it (as I said I'm currently a bit busy :D ).

**Comment by valeriano-manassero** [comment]
It looks a GH action issue to me. Will launch test pipeline later and hopefully we will be able to merge.

**Comment by lucacerone** [comment]
Thank you!

**Comment by lucacerone** [comment]
Just bumping a reminder for this PR :)

**Comment by valeriano-manassero** [comment]
Ok no worries, I was not aware I'm able to fix version directly on your branch; I updated version and now running CI tests.

**Pull request merged**